File: backend/pomoro-do/src/main/java/com/tico/pomoro_do/domain/auth/dto/response/TokenResponse.java
package com.tico.pomoro_do.domain.auth.dto.response;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;
import lombok.Getter;

@Getter
@Schema(description = "JWT Response")
public class TokenResponse {

    @Schema(description = "Access Token", nullable = false)
    private String accessToken;
    @Schema(description = "Refresh Token", nullable = false)
    private String refreshToken;

    @Builder
    public TokenResponse(String accessToken, String refreshToken) {
        this.accessToken = accessToken;
        this.refreshToken = refreshToken;
    }
}

File: Mobile/PomoroDo/app/src/main/java/com/tico/pomorodo/data/remote/models/response/TokenResponse.kt
package com.tico.pomorodo.data.remote.models.response

import com.tico.pomorodo.data.model.Token
import kotlinx.serialization.Serializable

@Serializable
data class TokenResponse(val accessToken: String, val refreshToken: String)

fun TokenResponse.toToken() = Token(accessToken, refreshToken)